.travis.yml

```diff
@@ -1,66 +1,58 @@
-dist: trusty
+---
 language: php
 
+php: "7.1"
+
+addons:
+  chrome: stable
+
 cache:
-  bundler: true
   directories:
     - "$HOME/.composer/cache"
     - "$HOME/.drush/cache"
-
-php:
-  - 7.1
+    - "$TMPDIR/phpstan/cache"
 
 env:
-  - VERSION=HEAD
-  # The final CI slot should be used to test the update path from the oldest
-  # available database fixture.
-  - VERSION=3.0.0
+  global:
+    - ORCA_SUT_NAME=acquia/lightning
+    - ORCA_SUT_BRANCH=8.x-3.x
+    - ORCA_VERSION=v1.0.0-alpha13
+    - ORCA_PACKAGES_CONFIG_ALTER=../lightning/tests/packages_alter.yml
+    - ORCA_FIXTURE_PROFILE=lightning
 
 matrix:
   fast_finish: true
-
-addons:
-  chrome: stable
+  include:
+    # Standard ORCA jobs.
+    - { name: "Static code analysis", env: ORCA_JOB=STATIC_CODE_ANALYSIS }
+    - { name: "Deprecated code scan", env: ORCA_JOB=DEPRECATED_CODE_SCAN_SUT }
+    - { name: "Isolated test w/ recommended package versions", env: ORCA_JOB=ISOLATED_RECOMMENDED }
+    - { name: "Integrated test w/ recommended package versions", env: ORCA_JOB=INTEGRATED_RECOMMENDED }
+    - { name: "Integrated test w/ recommended package versions & previous minor release of Drupal core", env: ORCA_JOB=CORE_PREVIOUS }
+    - { name: "Isolated test w/ dev package versions", env: ORCA_JOB=ISOLATED_DEV }
+    - { name: "Integrated test w/ dev package versions", env: ORCA_JOB=INTEGRATED_DEV }
+    - { name: "Integrated test w/ dev package versions & next minor dev version of Drupal core", env: ORCA_JOB=CORE_NEXT }
+    # Custom job.
+    - { name: "Upgrade test", env: ORCA_JOB=CUSTOM ORCA_CUSTOM_FIXTURE_INIT_ARGS="--dev --no-site-install" ORCA_CUSTOM_TESTS_RUN_ARGS="--sut-only" DB_FIXTURE=3.0.0 }
+  allow_failures:
+    - env: ORCA_JOB=ISOLATED_DEV
```

Comment:
I don't think ISOLATED_DEV should be allowed to fail. :)

Reply:
Removed.